scripts/src/main/resources/scripts/command/docker

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Initiates docker setup
+function doSetup() {
+  if doIsBatch
+  then
+    doFail "Interactive installation is required, cannot proceed in batch mode. Please rerun without batch option."
+  fi
+  if doIsWindows
+  then 
+    wsl_distro_list=$(wsl -l 2>/dev/null)
+    if [[ $wsl_distro_list == *"no installed distributions"* ]];then
+      doEcho "No Linux distributions found in WSL or WSL is disabled. Please enable WSL and install any linux distribution to proceed with docker setup."
+      exit 0
+    fi
```

Comment:
1. should we also echo a link to the devonfw-ide documentation with further instructions how to install WSL2 that we can even improve after the release?
2. we could move such "WSL" code to `functions` if we want to share it between `docker` and `kubernetes` commandlets.